<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mirall::DialogFolderChooser</class>
 <widget class="QDialog" name="Mirall::DialogFolderChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a Folder</string>
  </property>
  <widget class="QTreeView" name="treeViewFolderChooser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>511</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select a folder to backup:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>369</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="spinnerLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
